engine-runner-bin/README.md

```diff
@@ -0,0 +1,35 @@
+# Chainflip Engine (Runner)
+
+This README describes how the Chainflip Engine (CFE) actually runs and how the components interact to come together into a single runnable binary.
+
+The Chainflip Engine as it is run by a validator, consists of 3 build artefacts:
+
+1. The `enginer-runner` binary.
+2. A shared library of the old [engine-dylib](./../engine-dylib/). This is fetched from the previous release build when a release is built. Or directly from the repo ([`old-engine-dylib](./../old-engine-dylib/)) for localnets
```

Comment:
How does this fetch happen? Does cargo access github or somthing?

Reply:
Currently it's manually copy pasted - we can probably improve this, but for something rarely done, can be done later

Reply:
In that case I don't understand what 'fetched from the previous release build' means in this context. Is the plan that there will be a CI job that somehow updates this on github when we make a release?